src/Data/Recruit.ChangeScripts/050-vacancyReviews_migration.js

```diff
@@ -0,0 +1,7 @@
+{
+    print((new Date()) + "Start migrating documents from reviews to vacancyReviews collection.");
+
+    db.reviews.find().forEach( function(x){db.vacancyReviews.insert(x);} );
```

Comment:
What is to stop the `request rate` error, do you need a sleep?

Reply:
I have tried it on emulator and it worked without any errors, so I am hoping it would work. Note that InsertMany failed locally so I am assuming that local run is behaving similar to hosted.

Reply:
all you need is the change script without a sequence number prefix, it won't get run as part of deployment so I suggest using that as we will run this script hopefully just once and not part of deployment. After we have run the script then we can deploy the version of the code with the `MongoCollectionNames.cs` update.